core/src/main/java/org/apache/gravitino/storage/relational/service/MetadataObjectService.java

```diff
@@ -125,6 +142,8 @@ public static String getMetadataObjectFullName(String type, long metadataObjectI
         case SCHEMA:
           SchemaPO schemaPO = SchemaMetaService.getInstance().getSchemaPOById(objectId);
           if (schemaPO != null) {
+            // if fullName is null:
```

Comment:
I can't the point. What's the meaning of the comment.

Reply:
Sorry, I accidentally commited it, removed.